android/src/main/java/io/github/droidkaigi/feeder/notification/AppMessagingService.kt

```diff
@@ -0,0 +1,52 @@
+package io.github.droidkaigi.feeder.notification
+
+import android.util.Log
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import dagger.hilt.android.AndroidEntryPoint
+import io.github.droidkaigi.feeder.repository.DeviceRepository
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class AppMessagingService : FirebaseMessagingService() {
+
+    @Inject
+    lateinit var deviceRepository: DeviceRepository
+
+    private val serviceJob: Job = Job()
```

Comment:
Referencing implementations such as MainScope() and viewModelScope, SupervisorJob() may be better.
This does not need to be implemented in this PR.